docs/source/yelpsoa_configs.rst

```diff
@@ -609,6 +609,17 @@ Each Tron **action** of a job MAY specify the following:
     then the command should be something available in the docker container (it should NOT
     start with ``paasta local-run``).
 
+If a Tron **action** of a job is of executor type ``spark``, it MAY specify the following:
+
+  * ``spark_cluster``: The Paasta cluster on which to run spark jobs. Default to the same
+    cluster the tron job is running on.
+
+  * ``spark_pool``: The Paasta pool on which to run spark jobs. Default to ``default`` pool
+    if not specified.
+
+  * ``spark_args``: Dictionary of spark configurations documented in
```

Comment:
note that, as it currently stands, we dropped the initial `spark.` from the option name, and the code will prepend that before it sends the options over.  In other words, users will write this:

```
spark_args:
    cores.max: 10
    driver.port: 123456
```

not this:

```
spark_args:
    spark.cores.max: 10
    spark.driver.port: 123456
```

It's not actually clear to me that is the _best_ way to do things, it might just cause more confusion in the long run.  I'd be OK changing the code so it doesn't do this if you think it's better.

Reply:
My gut feeling is that we should keep the full names with the `spark.` prefix to prevent confusion.  Say, for instance, I am a spark n00b and I am looking at a Spark job and want to google the spark args -- better for them to be whole.

Reply:
Yeah, actually the code stripping `spark.` prefix has been removed so we are all good here.